### ISSUE TITLE
Stackoverflow Ads Research

### ISSUE THREAD

**Issue by marty-2015** [unikubehq/marketing#100]
- Targetierung
- Pricing

**Comment by marty-2015** [comment]
Update on Research @Truthans 

**Summary:**

- Ethical & Carbon Ads contacted (22.11.) for pricing.
- Stackoverflow from 15k per month
- Reddit Premium from 10k or Auction Ads (your ad competes with other ads in the same ZG); options Text, Image, Video, or Gallery. Next step: Create account with credit card.

My proposal: Starting with text ads on reddit.

**Comment by marty-2015** [comment]
Update Ethical Ads: 
- min. 500$, better 1.000$
- Narrow targeting on Kubernetes not possible
- Reporting only in impressions not in clicks

**Comment by marty-2015** [comment]
@Truthans check out my idea for "Promoted Post: Gallery Ad" Reddit

Image number 1:
![Reddit_GalleryAd_1200x1200_2](https://user-images.githubusercontent.com/86047585/144397247-e8e4aeb7-4a1c-4e22-89e0-e7923a78556d.png)

**Comment by marty-2015** [comment]
Image number 2:
![Reddit_GalleryAd_1200x1200_1](https://user-images.githubusercontent.com/86047585/144397392-7b06cbe5-ea0d-4c69-803b-aad1ec553e32.png)

**Comment by marty-2015** [comment]
Things to do:
- punchline, one sentence (hackday)
- green background
- set up a document


![Vorlage_Reddit_LinkAd_1200x628](https://user-images.githubusercontent.com/86047585/144837729-a1793f75-7d9e-43e1-90ea-5e1e559e78b7.png)

**Comment by Truthans** [comment]
Suggestion:
"Setting up a local K8s development cluster in 140 seconds without prior knowledge? No problem!"

**Comment by Truthans** [comment]
@Schille @SteinRobert considering that the rest of the team seems to have some sort of blockage when it comes to these marketing sentences, I'd hope that you may or may not have any more suitable suggestions for the one liner above.

**Comment by Schille** [comment]
I would not use the three of us giggling around. I personally don't want to see my face on an ad or in a pose that was not agreed on. 
We could instead use blurry images of "Kubernetes-structures" for the background and the unikube brand logo clearly visible beneath a bold (yet defensible) claim.

Brainstorming (target audience DevOps peeps):
- "Make your developers better Kubernetes developer ..."
- "Make your development team fail with Kubernetes from the beginning [...]"
- "Your development team doesn't know anything about Kubernetes? Fine. [...]"
- "You'd love to see complex service architectures successfully handled by your development teams in Kubernetes? Then look no further![...]"
- "Tired of explaining Kubernetes-things of your integration system to developers. Let them fail faster locally."

Brainstorming (target audience developers):
- "Tired of asking your grumpy DevOps people about Kubernetes-things? ..."
- "You are afraid of learning Kubernetes for your development tasks? Let unikube get the job done for you. [...]"
- "You are sick of learning Kubernetes for your development tasks? Let unikube get the job done for you. [...]"

more to come.

**Comment by marty-2015** [comment]
@Schille the images with the three of you were just the first draft, we already agreed on using the simpler version with the Unikube logo - like in the image above.

Concerning your suggestions on the marketing sentence: I like the first one **"Make your developers better Kubernetes developers"**, the rest is way too negative if you ask me. Failing with Kubernetes is not exactly a good marketing take on our product. What do you think @Truthans

**Comment by Schille** [comment]
I know it's sort of provoking. =)
However, based on our interviews I found that people that are already expert with Kubernetes tend to grasp the problem and Unikube as the solution much faster. Hence, I am feeling not too bad to trigger them with slightly provoking messages (with regard to their inexperienced coworkers). I am open to discussion, of course.

**Comment by Truthans** [comment]
@marty-2015 I actually don't think it's a terrible idea. "Failing early" in tech often also has a little more positive connotation. It plays along with i. e. Facebooks motto of "move fast and break things". Failing early means finding the right solution quicker and sparing others from you faults.

I would be inclined to try it. But I would try 2 or 3 versions anyway. In the end data will win.

**Comment by SteinRobert** [comment]
Hey guys, I would go a step further. Recently a tweet appeared in my timeline which looked like a person, linking their blog post about failing with X
So something like "Last week my K8S development setup broke - here's what I learned" or even "I keep failing at Kubernetes. It has to stop".

**Comment by marty-2015** [comment]
I did some more brainstorming on more provocative one-/two-liners, here are my ideas @Schille @SteinRobert @Truthans 

- Make your developers better at Kubernetes WITHOUT knowing Kubernetes."
- FCK Kubernetes! We have the right tool for you!
- Do you need a local Kubernetes development miracle? Here it is: Unikube!
- Everyone in your team asks you shit about Kubernetes. When you work with Unikube, you don’t have to face that shit any longer. Promised!
- Getting asked shit about Kubernetes all day long will end …when you start working with Unikube!
- Sick of learning Kubernetes? Unikube is the perfect tool for you!
- How would it feel to develop with Kubernetes, without having to know anything about it? EXCELLENT, right?!
- Do you want to feel like you rock Kubernetes? The only thing you need is Unikube!

**Comment by SteinRobert** [comment]
![image](https://user-images.githubusercontent.com/1231105/151507059-8d29543f-0b23-4ecb-83bb-b8d335a33063.png)

**Comment by Truthans** [comment]
I have refined a few of the suggestions:
"Your devs don't know much about Kubernetes? Fine."
"Fed up by devs asking you stuff about K8s? Make it stop."
"Learning Kubernetes stuff sucks, right? We feel you."

@marty-2015 let's discuss these in our meeting on Thursday.

**Comment by marty-2015** [comment]
V1 (grün)
![Reddit_LinkAd_1](https://user-images.githubusercontent.com/86047585/152355734-3bc80829-d563-4306-90ce-f418aadde2b6.png)

**Comment by marty-2015** [comment]
V1 (hellgrau)
![Reddit_LinkAd_1_hellgrau](https://user-images.githubusercontent.com/86047585/152355805-7dd03201-46d4-4618-b49e-e90e5187f6e3.png)

**Comment by marty-2015** [comment]
V2 (grün)
![Reddit_LinkAd_2_gruen](https://user-images.githubusercontent.com/86047585/152356889-619da261-b674-47c9-a355-9fd4bb8ad51a.png)

**Comment by marty-2015** [comment]
V2 (hellgrau)
![Reddit_LinkAd_2_hellgrau](https://user-images.githubusercontent.com/86047585/152356969-29ed5b5d-feb0-4b51-9132-0b9f6443fa36.png)

**Comment by marty-2015** [comment]
V3 (grün)
![Reddit_LinkAd_3_gruen](https://user-images.githubusercontent.com/86047585/152357008-84dc58c7-2923-4cf4-b6c3-4b829a64c8ad.png)

**Comment by marty-2015** [comment]
V3 (hellgrau)
![Reddit_LinkAd_3_hellgrau](https://user-images.githubusercontent.com/86047585/152357053-ae731f59-85fe-4619-a0f9-ac540a368494.png)

**Comment by marty-2015** [comment]
Please review @SteinRobert @Schille @Truthans

**Comment by Truthans** [comment]
@marty-2015 you did these in canva, right? Can you share them with me? If not, let's have a look together.

**Comment by marty-2015** [comment]
@Truthans no, I did those on gimp, because Canva doesnt have the right format and also I cannot save everything properly with our free account. I did them with gimp, you can find them here, I created an Ad folder in Unikube Marketing: https://drive.google.com/drive/u/1/folders/1wwzgeMvgUvcBG-4l8gZJnzll3uX3xp2K

**Comment by Truthans** [comment]
As discussed, try one version with the logo smaller and the text centrally aligend, both vertically and horizontally.

**Comment by marty-2015** [comment]
@Truthans like this?

V hellgrau:
![Reddit_LinkAd_2 1](https://user-images.githubusercontent.com/86047585/154970968-cb7e45e9-976b-403d-b9c4-20c9b899f1c3.png)

V grün:
![Reddit_LinkAd_2 2](https://user-images.githubusercontent.com/86047585/154970942-c5f75d4a-0c63-4c74-9ddd-baf9edc05a69.png)

**Comment by Truthans** [comment]
@marty-2015 I would put the "make it stop" in a new line and write DEVs capitalized. Other than that it looks good.

**Comment by marty-2015** [comment]
@Truthans ads are ready - 6 pngs, 3 different wordings in grey and green. You can find them here: https://drive.google.com/drive/u/0/folders/1wwzgeMvgUvcBG-4l8gZJnzll3uX3xp2K
